src/Kontent.Ai.ModelGenerator/Program.cs

```diff
@@ -68,40 +72,10 @@ public static async Task<int> Main(string[] args)
             }
         }
 
-        internal static IDictionary<string, string> GetSwitchMappings(string[] args)
+        private static void PrintSdkVersion(bool managementApi)
```

Comment:
I might use `CodeGeneratorOptions` as an argument (if we one time have i.t. GraphQL models, ...)

Reply:
And MAYBE we can have PrintSdkVersion as an Extension method, but this is just an idea.

Reply:
Cannot be done in a clear way, so we have decided to keep it here for now